react/src/components/AssetDetail/AssetButton.tsx

```diff
@@ -28,11 +28,16 @@ const AssetButton: React.FC<AssetButtonProps> = ({
   const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
   const featureType = getFeatureType(selectedFeature);
   const projectId = selectedFeature.project_id;
+  const [isSuccess, setIsSuccess] = useState<boolean>(false);
   const notification = useNotification();
   const featureId = selectedFeature.id;
   const { mutate: importFeatureAsset, isPending: isImporting } =
     useImportFeatureAsset(projectId, featureId);
 
+  useEffect(() => {
+    setIsSuccess(false);
+  }, [selectedFeature.id]);
```

Comment:
would this not work with the `isSuccess` value from the mutation instead?

Reply:
Not as it is no. The isSuccess key is the selected project feature list not the specific feature. Adding the feature id as an additional key won't update the list in the LoadedProject component. The loaded map project is looking for the feature id to change and since the import doesn't change an id, we are invalidating the key list features on import. The trickle down of selectedfeature state from loadedmapproject could possibly use some refactoring based on these conflicts